Source/Main.py

```diff
@@ -0,0 +1,8 @@
+import ImperatortoCK3Converter
+import Configuration
+
+ConfigurationFileName = input("Insert configuration filename: ")
```

Comment:
it's always configuration.txt, as provided by the frontend. No need to prompt anything.